### ISSUE TITLE
Pin tables organized by peripheral modes?

### ISSUE THREAD

**Issue by jadonk** [jadonk/beaglebone-ai-wiki#2]
Would it make more sense to document only modes you can put together a valid interface? There are several modes where there are only parts of a peripheral exposed and not enough to get anything working?

**Comment by cathywicks** [comment]
Yes, I agree that it makes more sense to only document those that have complete pins available for a function.
